string entity_id

geometry_msgs/Pose  pose
geometry_msgs/Twist velocity
geometry_msgs/Accel acceleration

float64 length        # in meters
float64 width         # in meters
float64 height        # in meters (can be 0)

